build.gradle.kts

```diff
@@ -11,7 +11,7 @@ repositories {
 }
 
 plugins {
-    id("org.jetbrains.intellij").version("1.6.0")
```

Comment:
The test task is broken with 1.8.0 and/or Gradle 7.5.x.
Java 11, Gradle 7.4.2 and plugin 1.6.0 works.
I think you also run into https://github.com/detekt/detekt-intellij-plugin/issues/211

Reply:
Ahhhhhhh damn. Let me roll back Gradle to 7.4.2 and see if that's enough :)

Reply:
Yep it's something with Gradle 7.5, rolled back to 7.4.2 and it works. Can keep using the IJ plugin 1.8.0.